CMSgov/cms-carts-seds
# MDCT-223: Fix Dead Spots in Header Menu (main)

# Description

Same as `master`. 

Closes [MDCT-223](https://qmacbis.atlassian.net/browse/MDCT-223). Users have trouble finding the clickable area of the CARTS header drop-down menu. Now it should be clickable all-around. Also, changed Helpdesk to Contact Us based on Design's suggestion.

## How to test

Populate `.env` files in root and **ui-src**.

`nvm use`
`./dev local`

Navigate to `localhost:3000` and log in as any user type. On the top-right, verify that the menu items are clickable.

## Dependencies

N/A

# Get it done

Clicks!

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
